README.md

```diff
@@ -3,6 +3,19 @@ PHP Models for the OpenActive Opportunity and Booking Specifications
 
 OpenActive aims to provide model files for all classes defined in its Opportunity and Booking specifications across the PHP, Ruby, and .NET languages. This repository is intended for the PHP files; see also the [Ruby](https://github.com/openactive/models-ruby/) and [.NET](https://github.com/openactive/OpenActive.NET) implementations.
 
+## Table of Contents
+- [Requirements](#requirements)
+- [Installation](#installation)
+- [Usage](#usage)
+    - [Models](#models)
+        - [OpenActive](#openactive)
+        - [Schema.org](#schemaorg)
+    - [RPDE](#rpde)
+    - [Enums](#enums)
+    - [Serialization](#serialization)
+        - [`serialize($obj, $prettyPrint = false)`](#serializeobj-prettyprint--false)
+        - [`deserialize($data)`](#deserializedata)
+
 ## Requirements
 This project requires PHP >=5.6.
 While most of the functionality should work down to PHP 5.4, some functionality (especially around parsing of offset for DateTimeZone) will not work with that version of PHP (see the [DateTimeZone PHP docs](https://www.php.net/manual/en/datetimezone.construct.php#refsect1-datetimezone.construct-changelog) for more info).
```

Comment:
Can we suggest a solution to this? What should they do or not do different if on 5.4?

Reply:
At the moment the work-around for a library user would be not to specify timezone offsets in their datetime string representations. If you are good with this I can add to docs.

Reply:
So use UTC instead? Sure - that's great if you could add that with perhaps a clear example?

Reply:
(noting that their data will likely be coming from a database, so perhaps the guidance should cover how to deal with offsets that are coming from MySQL?

Reply:
It would technically use whatever timezone is defined, either in the server's `php.ini` configuration, or at some point in the application/script ([date_default_timezone_set](https://www.php.net/manual/en/function.date-default-timezone-set.php)).

Nothing is technically stopping a PHP developer to set different timezones for different scripts across the same app, although I can't think of any valid reason.

Reply:
oh sorry so it's not inaccurate data that's the issue (it won't read the time wrong on input?), it's just that it defaults the output timezone? In that case please do make that clear so that implementers know this isn't a major issue